e2e-polybft/txpool_test.go

```diff
@@ -159,6 +161,36 @@ func TestE2E_TxPool_Transfer_Linear(t *testing.T) {
+
+	customArtifact, err := artifact.DecodeArtifact([]byte(smJSON))
+	require.NoError(t, err)
+
+	sidechainKey, err := wallet.GenerateKey()
+	require.NoError(t, err)
+
+	cluster := framework.NewTestCluster(t, 4,
+		framework.WithPremine(types.Address(sidechainKey.Address())),
+	)
+	defer cluster.Stop()
+
+	require.NoError(t, cluster.WaitForBlock(5, 30*time.Second))
```

Comment:
Is it enough here to wait for just a single block?

Reply:
https://github.com/0xPolygon/polygon-edge/pull/1140/commits/cb1fa0a0415784fa391b7dd3961651021f9d934a